pizza.txt

```diff
@@ -9,3 +9,15 @@ sauce:
 Salt and pepper to taste
 Red pepper flakes (optional, for a bit of heat)
 
+Pizza Dough Ingredients:
+2 ¼ teaspoons (1 packet) active dry yeast
+1 ½ cups warm water (110°F or 45°C)
+3 ½ to 4 cups all-purpose flour
+2 tablespoons olive oil
+1 teaspoon sugar
+2 teaspoons salt
+
+Crust:
+Knead the dough for 8 minutes.
+Stretch it into a flat circle.
+Add some seasoning if you like.
```

Comment:
isn't this already on main?

Reply:
Yes, it's already on main. I don't know why this commit is on this PR?